README.md

```diff
@@ -346,7 +346,7 @@ Every API returns a Promise but also has a corresponding API with 'Sync' on the
 | [getTotalMemory()](#gettotalmemory)                               | `Promise<number>`   |  ✅  |   ✅    |   ❌    |
 | [getUniqueId()](#getuniqueid)                                     | `string`            |  ✅  |   ✅    |   ✅    |
 | [getUsedMemory()](#getusedmemory)                                 | `Promise<number>`   |  ✅  |   ✅    |   ❌    |
-| [getUserAgent()](#getuseragent)                                   | `Promise<string>`   |  ✅  |   ✅    |   ❌    |
+| [getUserAgent()](#getuseragent)                                   | `string`            |  ✅  |   ✅    |   ❌    |
```

Comment:
getUserAgent is async and returns a promise https://github.com/react-native-community/react-native-device-info/blob/master/src/index.ts#L363

Reply:
Oh, my mistake haha, I'm using a older version.. :/ Sorry..
Fixing...